index.html

```diff
@@ -23,7 +23,8 @@ <h2>Participants</h2>
         <li><span>Janarthan</span> Vijayakumar</li>
         <li><span>Vithursanan</span> Ananthakumar</li>
         <li><span>Partheepan</span> Vasanthakumar</li>
+        <li><span>Sorubaraj</span> Ravinthiraraj</li>
```

Comment:
Hi Soruba Your changes are acceptable. But
1.  It will take time to add you.
2.  Please wait and update your changes after 2 days.